cpp/src/search/search.cu

```diff
@@ -210,6 +212,88 @@ bool contains(column_view const& col,
                                              col, value,
                                              stream, mr);
 }
+
+template <typename Element>
+void multi_contains_compute(column_view const& col,
+                            column_view const& in_col,
+                            mutable_column_view &result_view,
+                            cudaStream_t stream = 0) {
+
+  auto hash_set = fixed_hash_set<Element>::create(in_col, stream);
+  auto device_hash_set = hash_set.to_device();
+
+  thrust::transform(rmm::exec_policy(stream)->on(stream),
+                    col.begin<Element>(),
+                    col.end<Element>(),
+                    result_view.begin<bool8>(),
+                    [device_hash_set] __device__ (Element e) {
+                      return device_hash_set.contains(e);
+                    });
+}
+
+template <>
+void multi_contains_compute<string_view>(column_view const& col,
```

Comment:
I would not think this specialization would be necessary.

Reply:
Perhaps there's a better way to do this.  The fixed width elements allow me to just iterate over the range of elements in the transform call.  The best way I could find to do the same for strings is to create the string_vector_from_column.  Is there another way to do this?

Reply:
Nevermind... found a better way.